ext/plugins/base.js

```diff
@@ -41,8 +41,6 @@ module.exports = function(addonContext, name) {
         // process `<!-- template-lint foo=bar -->` comments
         if (bodyEntry.type === 'CommentStatement' && bodyEntry.value.indexOf('template-lint') === 1) {
           this._processConfigNode(bodyEntry);
-          // remove the entry
-          node.body.splice(i, 1);
```

Comment:
I agree, but we don't really have a choice. If we do remove like this was doing before then the configuration only works for the first plugin (and each of them need this config).